chapter-6/scratchpad.ts

```
@@ -0,0 +1,253 @@
+type ExistingUser = {
+    id: number;
+    name: string;
+};
+
+// A new user that hasn't been saved to the server yet
+type NewUser = {
+    name: string;
+};
+
+function deleteUser(user: { id?: number; name: string }) {
+    delete user.id;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)